### PR TITLE
feat: Adds new APIs on frontend to use in onboarding view

### PR DESCRIPTION
**Pull request by nithindavid**
## Description

New API endpoint support for bulk email invite and account setup

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/v3/api/auth.js [hunk 1]
```diff
@@ -42,6 +42,19 @@ export const register = async creds => {
   return null;
 };
 
+export const accountSetup = async ({ id, name, locale }) => {
+  try {
+    const response = await wootAPI.put(`api/v2/accounts/${id}.json`, {
```
Comment (pranavrajs) [inline app/javascript/v3/api/auth.js:47]: Why do we have `.json` added here? Is there a specific reason?

File: app/javascript/v3/api/auth.js [hunk 2]
```diff
@@ -42,6 +42,19 @@ export const register = async creds => {
...
+      name,
+      locale,
+    });
+    return response.data;
+  } catch (error) {
+    throwErrorMessage(error);
+  }
+  return null;
```
Comment (pranavrajs) [inline app/javascript/v3/api/auth.js:55]: I guess this statement would never be executed.

File: app/javascript/dashboard/api/agents.js [hunk 3]
```diff
@@ -1,9 +1,17 @@
+/* global axios */
+
 import ApiClient from './ApiClient';
 
 class Agents extends ApiClient {
   constructor() {
     super('agents', { accountScoped: true });
   }
+
+  bulkInvite({ emails }) {
```
Comment (pranavrajs) [inline app/javascript/dashboard/api/agents.js:10]: Please add it to the spec.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Minor comments, please take a look.

File: app/javascript/v3/api/auth.js (hunk 2 above)
Comment (nithindavid) [reply app/javascript/v3/api/auth.js]: Makes sense. Removed it.

File: app/javascript/dashboard/api/agents.js (hunk 3 above)
Comment (nithindavid) [reply app/javascript/dashboard/api/agents.js:10]: Added spec

File: app/javascript/v3/api/auth.js (hunk 1 above)
Comment (nithindavid) [reply app/javascript/v3/api/auth.js]: Removed the function as we are not using it.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**